<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoScribeMainWindow</class>
 <widget class="QMainWindow" name="NoScribeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoScribe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="sidebar">
      <layout class="QVBoxLayout" name="verticalLayoutSidebar">
       <item>
        <widget class="QPushButton" name="btnAudioFile">
         <property name="text">
          <string>Audio File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTranscriptFile">
         <property name="text">
          <string>Transcript File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtLog"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
